spec/pdfs/response_report_spec.rb

```diff
@@ -33,7 +33,8 @@ def complete_response
       expect(page_1).to include('In de onderstaande tabel ziet u de resultaten van de ingevulde vragenlijst.')
     end
 
-    it 'contains the date of the response' do
+    it 'contains the date of the request and response' do
+      expect(page_2).to include('Aangevraagd op')
```

Comment:
Lijkt me wel aardig om in deze spec (en de Rails viewspec) ook te valideren dat de datum exact is wat je verwacht, om zeker te zijn dat het tijdzoneprobleem niet terugkomt.